tg_forwarder_client.py

```diff
@@ -28,12 +33,27 @@ def get_settings() -> Settings:
     return Settings()
 
 
-async def channel_handler(_, message):
+def send_media_group(client, media_group_id):
+    settings = get_settings()
+
+    for chat_id in settings.chats:
+        loop.create_task(client.forward_messages(chat_id, *queues[media_group_id]))
+
+
+async def channel_handler(client, msg):
     settings = get_settings()
-    if message.chat.username in settings.channels and not message.edit_date:
-        logging.info(message.chat)
-        for chat_id in settings.chats:
-            await message.forward(chat_id)
+    if msg.chat.username in settings.channels and not msg.edit_date:
+        logging.info(msg.chat)
+
+        if hasattr(msg, 'media_group_id') and msg.media_group_id:
+            if msg.media_group_id not in queues:
+                queues[msg.media_group_id] = (msg.chat.id, [])
+                loop.call_later(10, send_media_group, client, msg.media_group_id)
+
+            queues[msg.media_group_id][1].append(msg.message_id)
+        else:
+            for chat_id in settings.chats:
+                await msg.forward(chat_id)
```

Comment:
```suggestion
        return
    for chat_id in settings.chats:
        await msg.forward(chat_id)
```

Борьба с else во имя добра и света